src/boot/TranslationAPIProvider.js

```diff
@@ -0,0 +1,28 @@
+/* @flow */
+import { PureComponent } from 'react';
+import { injectIntl, intlShape } from 'react-intl';
+
+import type { ChildrenArray } from '../types';
+
+type Props = {
+  children?: ChildrenArray<*>,
+  intl: intlShape.isRequired,
+};
+
+class TranslationAPIProvider extends PureComponent<Props> {
+  static childContextTypes = {
+    _: string => {},
```

Comment:
`string => {},` is not correct here - it defines a function with an argument **named** `string`. This should not reflect the function signature, it is a placeholder that we use in place of PropTypes. So just `() => {}` is enough.

Do not forget to update the `types.js` => `type Context` with the `_` value.

Reply:
Yeah, that was a typo, and in this I'm returning a string so I changed to `() => '' `

Reply:
Let's keep this `() => {}` like everywhere else -- these lines are still puzzling to me :smile: , but there's no point in making it different in a way that we don't have a good story for why it'd help anything.